config/sync/user.role.person.yml

```
@@ -3,10 +3,15 @@ langcode: da
 status: true
 dependencies:
   module:
+    - administerusersbyrole
     - profile
 id: person
 label: Person
 weight: 2
 is_admin: false
 permissions:
+  - 'cancel users by role'
+  - 'cancel users with role person'
```

This will allow users with the role `person` to cancel other persons' accounts. The right permission to add must be `cancel account` (which actually means “cancel own account").

```suggestion
  - 'cancel account'
```

(the dependency on `administerusersbyrole` can then be removed as well)